src/laplace.jl

```diff
@@ -249,9 +249,9 @@ function optimize_prior!(
 )
 
     # Setup:
-    logP₀ = isnothing(λinit) ? log.(unique(diag(la.P₀))) : log.([λinit])   # prior precision (scalar)
-    logσ = isnothing(σinit) ? log.([la.σ]) : log.([σinit])                 # noise (scalar)
-    opt = Adam(lr)
+    logP₀ = isnothing(λinit) ? log.(unique(diag(la.P₀))) : log.([λinit])     # prior precision (scalar)
+    logσ = isnothing(σinit) ? log.([la.σ]) : log.([σinit])                   # noise (scalar)
+    opt = Adam(lr)                                                           # Adam is a stochastic gradient descent (SGD) optimization algorithm, using lr as learning rate
```

Comment:
On our discussion yesterday: Adam is applicable to mini-batch GD

Reply:
Yes, fixed it.